Makefile

```diff
@@ -548,7 +548,7 @@ test_e2e_xdc: bins
 		go test $(TEST_ARG) -coverprofile=$@ "$$dir" $(TEST_TAG) | tee -a test.log; \
 	done;
 
-cover_profile: bins
```

Comment:
bins may be excessive, but this is broadly useful to ensure it builds before running all tests.  otherwise it can get like 90% of the way through before failing on a syntax error or partial-refactor or something.

Reply:
We have a separate job that generates bins and verifies copyright and etc. If something is broken, I expect tests to fail. In this case bins generation/verification takes a lot of time.

Reply:
yea, fair.  I suppose this is really only intended for CI, most people should be using `make cover` instead?